src/js/angular/resources/chart-models/chart-data.js

```diff
@@ -51,25 +55,16 @@ export class ChartData {
         });
     }
 
-    /**
-     * Must implement
-     * @return {string} the title name
-     */
-    getTitle() {
-        return 'No chart title set';
-    }
-
-    setTitle(title) {
+    setSubTitle(keyValues) {
         this.chartOptions.title.enable = true;
-        this.chartOptions.title.text = title;
-    }
-
-    getSubTitle() {
-    }
-
-    setSubTitle(subTitle) {
-        this.chartOptions.subtitle.enable = true;
-        this.chartOptions.subtitle.html = subTitle;
+        const subTitle = keyValues.map((keyValue) => {
+            let values = keyValue.value;
+            if (Array.isArray(keyValue.value)) {
+                values = keyValue.value.join('/');
+            }
+            return keyValue.label + (keyValue.value !== undefined ? `<span class="data-value">${values}</span>` : '');
```

Comment:
This `keyValue.value !== undefined ? ...` can be just `values ? ...` or `values.length ? ...`

Reply:
added additional logic to return from the method if `value` is array and its length is 0